packages/feedback/src/core/sendFeedback.ts

```diff
@@ -1,100 +1,65 @@
-import { createAttachmentEnvelope, createEventEnvelope, getClient, withScope } from '@sentry/core';
-import type { FeedbackEvent, SendFeedback, SendFeedbackParams } from '@sentry/types';
+import { captureFeedback } from '@sentry/core';
+import { getClient } from '@sentry/core';
+import type { SendFeedback, SendFeedbackParams, TransportMakeRequestResponse } from '@sentry/types';
+import type { Event } from '@sentry/types';
 import { getLocationHref } from '@sentry/utils';
-import { FEEDBACK_API_SOURCE, FEEDBACK_WIDGET_SOURCE } from '../constants';
-import { prepareFeedbackEvent } from '../util/prepareFeedbackEvent';
+import { FEEDBACK_API_SOURCE } from '../constants';
 
 /**
  * Public API to send a Feedback item to Sentry
  */
 export const sendFeedback: SendFeedback = (
```

Comment:
Doesn't have to be in this PR, but should we continue to export this publicly? Or just leave it as an internal helper for our own UI?

Reply:
that's a good question, I don't have a strong feeling there! I _think_ for backwards compatibility we might as well continue to export it, but I'll defer to whatever you folks think about this 🤔

Reply:
I think with the new `captureFeedback()` being a thing, this isn't too important for people anymore. If we were going to remove it now would be a fine time. 

It seems like if we keep it, then the docs will be something like:
> There is `captureFeedback()`, and also `sendFeedback()`... the differences are....

OR if we remove it, docs could instead mention:
> There is `captureFeedback()`. You can response by doing something like `client.on('afterSendEvent', (event, response) => ...)` and check that `event.event_id === eventId` 

I'm starting a migration doc to track moving from "feedback beta & v7" into "v8 & GA", so this could fit there too.
https://github.com/getsentry/sentry-javascript/pull/11731

Reply:
I'll remove the public export then in a follow up PR 👍